tests/Type/ValidationTest.php

```diff
@@ -153,7 +154,10 @@ private function withModifiers(array $types): array
                 $types
             ),
             array_map(
-                static fn (Type $type): NonNull => Type::nonNull($type),
+                static function (Type $type): NonNull {
+                    /** @var Type&NullableType $type */
```

Comment:
shouldn't this be fn @param instead?

Reply:
I would prefer it, but PHPStan does not consider `@param` in closures.

Reply:
Oh right, https://github.com/phpstan/phpstan/issues/3770

Maybe we can add the param as well so we get notified to remove @var when phpstan is fixed.

Reply:
I subscribed to that issue.